src/async-token.ts

```diff
@@ -3,10 +3,7 @@ import type { AxiosInstance } from './types'
 export default function asyncToken(instance: AxiosInstance, getToken: () => Promise<string>): void {
   instance.interceptors.request.use(function (config) {
     return getToken().then((accessToken) => {
-      config.headers = {
-        ...config.headers,
```

Comment:
Just curious why you changed this. Seems like a change in logic 🤔

Reply:
Instead of using object notation, we use the methods on the Headers object to define the `Authorization` header.
Technically, this should not make any difference, but since the typing is stronger, we now exclusively get a Headers object here.